ruby/CRUD-examples/delete/extra_fun_1.rb

```diff
@@ -0,0 +1,42 @@
+# Code Sample from the tutorial at https://learncodeshare.net/2016/11/09/delete-crud-using-ruby-oci8/
+#  section titled "Extra Fun 1"
+# Using the base template, the example code executes a simple delete using named bind variables.
+
+require 'oci8'
+connectString = ENV['DB_CONNECT'] # The environment variable for the connect string: DB_CONNECT=user/password@database
+
+def get_all_rows(label, data_type = 'people')
+  connectString = ENV['DB_CONNECT'] # The environment variable for the connect string: DB_CONNECT=user/password@database
```

Comment:
The get_all_rows functions do not have access to the top connectString variable.  I moved the top declaration down near where it's used.  I agree this looks cleaner.